src/components/MerchantsPage/gam/CampaignListItem.tsx

```diff
@@ -107,12 +101,11 @@ const CampaignListItem = (props: Props) => {
             )}
           </Description>
           <CampaignProgressBar
+            endDate={campaign.end_date}
```

Comment:
probably could just pass a campaign through at this point but idt it really matters